src/api/spec/controllers/webui/comments_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+RSpec.describe Webui::CommentsController, type: :controller do
+  let(:user) { create(:confirmed_user, login: 'luck') }
+
+  describe 'POST #create' do
+    let(:project) { create(:project) }
+    let(:package) { create(:package) }
+    let(:bs_request) { create(:bs_request) }
+
+    before do
+      login user
+    end
+
+    context 'with a valid comment' do
+      RSpec.shared_examples 'saving a comment' do
+        before do
+          params = { comment: { body: "This #{commentable.model_name.singular} is AWESOME!" },
+                     commentable_type: commentable.class, commentable_id: commentable.id }
+          post :create, params: params
+        end
+
+        it 'should have flash notice' do
```

Comment:
Why not also one liner?

``
it { expect(flash[:notice]).to eq('Comment was successfully created.') }
``